src/main/java/com/minecolonies/blockout/loader/json/JsonPaneParams.java

```diff
@@ -0,0 +1,184 @@
+package com.minecolonies.blockout.loader.json;
+
+import com.google.common.collect.ImmutableList;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.minecolonies.blockout.loader.IPaneParams;
+import com.minecolonies.blockout.util.Localization;
+import com.minecolonies.blockout.views.View;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+public class JsonPaneParams implements IPaneParams
+{
+    private final JsonObject object;
+    private       View       parent;
+
+    public JsonPaneParams(final JsonObject object) {this.object = object;}
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 9. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)